yolox/data/datasets/coco.py

```diff
@@ -65,8 +65,8 @@ def __init__(
         remove_useless_info(self.coco)
         self.ids = self.coco.getImgIds()
         self.class_ids = sorted(self.coco.getCatIds())
-        cats = self.coco.loadCats(self.coco.getCatIds())
-        self._classes = tuple([c["name"] for c in cats])
+        self.cats = self.coco.loadCats(self.coco.getCatIds())
```

Comment:
why `cats` becomes attribute here?

Reply:
`cats` was changed to an attribute so that the category names can be accessed by other functions.